default_config.yaml

```diff
@@ -1,7 +1,7 @@
 path:
-  seechange_data: null
+  data_root: null
   data_temp: null
-
+  server_data: null
```

Comment:
See comment in README re: storing server location.

Rather than having any archive server information in the config at all, we should just have relative file paths.  data_root says where to find it on the local system, but on the archive, the system we put up to connect to the archive should be responsible for figuring out any absolute paths.  (It may also end up being more complicated than that, as we might archive old images to HPSS or some such.) 

There should be some kind of overall "system" or "collection" variable, so that the same server can handle multiple different sets of images.  E.g., with curveball, I have a "dev" overall collection where I muck about.  There's also a "emily" collection where Emily does her dev work.  Then there's a "ztf" collection where Autumn and I are building lightcurves from ztf, etc., and that's a production collection.  All of these connect to the same server for storing them in the archive on NERSC; server figures out where the root directory is based on the collection it gets, and then we have relative filepaths after that.